### PR TITLE
Feature/drive

### PR DESCRIPTION
**Pull request by notjasonl**

### CODE REVIEW

File: src/main/kotlin/com/team4099/robot2020/subsystems/Drive.kt [hunk 1]
```diff
@@ -270,23 +270,23 @@ object Drive : Subsystem {
...
-                signal.rightMotor * Constants.Drive.MAX_RIGHT_OPEN_LOOP_POWER
-        )
     }
 
     @Synchronized
     private fun setLeftRightPower(left: Double, right: Double) {
-        leftMasterTalon.set(ControlMode.PercentOutput, left)
-        rightMasterTalon.set(ControlMode.PercentOutput, right)
+        leftMasterTalon.set(ControlMode.PercentOutput, left, DemandType.ArbitraryFeedForward, 0.0)
+        rightMasterTalon.set(ControlMode.PercentOutput, right, DemandType.ArbitraryFeedForward, 0.0)
```
Comment (atzho) [inline src/main/kotlin/com/team4099/robot2020/subsystems/Drive.kt:300]: This us more of a workaround than a fix, make sure the setter in lazytalon works instead

File: src/main/kotlin/com/team4099/robot2020/Robot.kt [hunk 2]
```diff
@@ -97,8 +98,10 @@ object Robot : TimedRobot() {
             disabledLooper.stop() // end DisabledLooper
             enabledLooper.start() // start EnabledLooper
 
-            autoModeExecuter = AutoModeExecuter(DashboardConfigurator.getSelectedAutoMode())
+//            autoModeExecuter = AutoModeExecuter(DashboardConfigurator.getSelectedAutoMode())
+            autoModeExecuter = AutoModeExecuter(DriveForwardMode(0.0))
```
Comment (atzho) [inline src/main/kotlin/com/team4099/robot2020/Robot.kt:108]: Should go back to dashboardconfig selected mode

**Review by rbhog — APPROVED** [review]
Nice job??

File: src/main/kotlin/com/team4099/robot2020/subsystems/Drive.kt [hunk 3]
```diff
@@ -447,8 +447,8 @@ object Drive : Subsystem {
             enterVelocityClosedLoop()
             currentState = DriveControlState.MOTION_MAGIC
         }
-        leftMasterTalon.set(ControlMode.MotionMagic, metersToNative(leftMeters))
-        rightMasterTalon.set(ControlMode.MotionMagic, metersToNative(rightMeters))
+        leftMasterTalon.set(ControlMode.MotionMagic, metersToNative(leftMeters), DemandType.ArbitraryFeedForward, 0.0)
```
Comment (atzho) [inline src/main/kotlin/com/team4099/robot2020/subsystems/Drive.kt:461]: Use actual fix pls i am adding so review get angwy

**Review by atzho — CHANGES_REQUESTED** [review]

**Review by rbhog — APPROVED** [review]
Nice Job!

**Pull request merged**